lib/streamlit/elements/media_proto.py

```diff
@@ -88,7 +88,7 @@ def _marshall_av_media(proto, data, mimetype):
     elif isinstance(data, io.BytesIO):
         data.seek(0)
         data = data.getvalue()
-    elif isinstance(data, io.RawIOBase):
+    elif isinstance(data, io.RawIOBase) or isinstance(data, io.BufferedReader):
```

Comment:
Is this related to this PR? What does it do?

Reply:
Oh, it fixes that bug in st.audio that people found last week.